uportal-war/src/main/java/org/jasig/portal/rendering/StructureAttributeSource.java

```diff
@@ -30,8 +32,15 @@
  * @version $Revision$
  */
 public class StructureAttributeSource extends StylesheetAttributeSource {
+    private static final Logger LOGGER = LoggerFactory.getLogger(StructureAttributeSource.class);
```

Comment:
Even further, since the class is not final, I suggest doing `protected` so that children classes can use it :)